dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/hooks/PreCheckExistenceValidationHookTest.java

```diff
@@ -123,8 +126,8 @@ public void setUp()
         when( bundle.getProgramInstance( ENROLLMENT_UID ) ).thenReturn( getEnrollment() );
         when( bundle.getProgramStageInstance( SOFT_DELETED_EVENT_UID ) ).thenReturn( getSoftDeletedEvent() );
         when( bundle.getProgramStageInstance( EVENT_UID ) ).thenReturn( getEvent() );
-        when( preheat.getRelationship( getPayloadRelationship() ) )
-            .thenReturn( getRelationship() );
+        when( preheat.getRelationship( RELATIONSHIP_UID ) ).thenReturn( getRelationship() );
+        when( preheat.getRelationship( SOFT_DELETED_RELATIONSHIP_UID ) ).thenReturn( softDeletedRelationship() );
```

Comment:
Could you please add the mock to the test needing it.

I try to remove `@MockitoSettings( strictness = Strictness.LENIENT )` when I can in our tests. This helps us keeping them clean (only needed mocks/stubs) in the long run.

If you have the energy and time you could 🧽 this test class and get rid of `@MockitoSettings( strictness = Strictness.LENIENT )` 😋 Or in a later PR, so we get this merged quicker